modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+| "northwind-2022" | {index: 3, newestnorthwind: TRUE} |
++------------------------------------------------------+
+----
+
+==== Create database aliases with the same name as an existing alias
+
+Adding a local database alias with the same name as an existing local or remote alias does nothing with the `IF NOT EXISTS` clause but fails without it.
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` IF NOT EXISTS FOR DATABASE `northwind-graph-2020`
+----
+
+==== Create or replace database aliases
+
+It is also possible to replace a database alias.
+The old alias may be either local or remote.
+
+.Query
+[source, cypher]
+----
+CREATE OR REPLACE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`
+----
+
+This is equivalent to running the following two queries consecutively:
+
+.Query
+[source, cypher]
+----
+DROP ALIAS `northwind` IF EXISTS FOR DATABASE
+----
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`
+----
+
+[[database-management-create-remote-database-alias]]
+=== Create remote database aliases
+
+A database alias can target a remote database by providing an URL and the credentials of a user on the remote Neo4j DBMS.
+See link:{neo4j-docs-base-uri}/operations-manual/{page-version}/manage-databases/remote-alias[Connecting remote databases] for the necessary configurations.
+
+ remote database aliases also allows `IF NOT EXISTS` and `OR REPLACE` clauses.
+Both check for any remote or local database aliases.
+
+
+==== Create remote database aliases
+
+//Skip testing all remote examples because it requires a lot of configuration, both server and client side.
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-northwind` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+USER alice
+PASSWORD 'example_secret'
+----
+
+When you create a database alias pointing to a remote database, its details can be shown with the `SHOW ALIASES FOR DATABASE` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `remote-northwind`
+FOR DATABASE
+----
+
+.Result
+[role="queryresult"]
+----
++----------------------------------------------------------------------------------------------------------+
+| name               | composite | database               | location | url                       | user    |
++----------------------------------------------------------------------------------------------------------+
+| "remote-northwind" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "alice" |
++----------------------------------------------------------------------------------------------------------+
+----
+
+
+==== Create remote database aliases with driver settings
+
+It is possible to override the default driver settings per database alias, which are used for connecting to the remote database.
+
+This is the list of the allowed driver settings for remote database aliases:
+
+.dbms.routing.driver.ssl_enforced
```

Comment:
I'm gonna assume you have checked that this is the correct full names of the settings